specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -1434,6 +1434,13 @@
           "items": {
             "$ref": "#/definitions/TextLine"
           }
+        },
+        "selectionMarks": {
+          "description": "List of selection marks extracted from the page.",
```

Comment:
It would be great to define what a selection mark is - i.e. is an example a box that has been checked?